opendrift/readers/basereader/consts.py

```diff
@@ -6,11 +6,11 @@
     'y_wind': {'valid_min': -50, 'valid_max': 50, 'units': 'm/s',
         'long_name': 'Component of wind along y-direction (northwards if '
                       'projection is lonlat/Mercator)'},
-    'x_sea_water_velocity': {'valid_min': -10, 'valid_max': 10,
+    'x_sea_water_velocity': {'valid_min': -20, 'valid_max': 20,
```

Comment:
ok not too sure how to do that

Reply:
I have tried to restore the original state with checkout -- Tell me if it looks better

Reply:
I see that the two unused file were deleted, which is good. There is one conflict due to the valid_min/max-limits, but I believe this can be resolved in the browser as seen below (but probably this has to be done by yourself?).

Then we also need to find a solution for installation without users needing to set environment variables. But it could be that this environment variable is not needed anyway(?)

Eventually we would also like to have a unit-test for this reader, though then one would need a stable thredds server with an available dataset. However, in the meantime, a placeholder test could be made to simply check that it works well to import this reader and the selafin-tool.

Reply:
ok I don't really know how to discard my version. 
regarding the environmental variables they are needed in Telemac to run the python scripts and other things. It would not run without it. 
For the test I have something from Telemac itself but since it has no coordinate and I wasn't sure about how to setup proj4 in that case, I gave up

Reply:
Done it I think